x/mock/simulation/params.go

```diff
@@ -41,6 +44,29 @@ type Params struct {
 	BlockSizeTransitionMatrix TransitionMatrix
 }
 
+// getBlockSize returns a block size as determined from the transition matrix.
+// It targets making average block size the provided parameter. The three
+// states it moves between are:
+//  - "over stuffed" blocks with average size of 2 * avgblocksize,
+//  - normal sized blocks, hitting avgBlocksize on average,
+//  - and empty blocks, with no txs / only txs scheduled from the past.
+func getBlockSize(r *rand.Rand, params Params,
```

Comment:
I don't think this belongs in params, perhaps put in utils.go?

Reply:
Yeah okay - I've moved it - originally my intention during this refactor was to make getBlockSize a property of params, but that doesn't seem quite right unless params is expanded or a new struct is introduced. Anyways I think that change is beyond the scope of this PR so moving to utils.go is good for now